src/taco-cli/cli/kit.ts

```diff
@@ -436,13 +438,15 @@ class Kit extends commands.TacoCommandBase {
             // Remove all the updatable plugins and re-add them
             var pluginOpts: Cordova.ICordovaPluginOptions = { save: true };
             return CordovaWrapper.plugin("remove", null, Object.keys(components))
-                .then(function(): Q.Promise<any> {
+                .then(function (): Q.Promise<any> {
                     // Do not save in the case of updating to CLI project
                     pluginOpts.save = editParams.isKitProject;
                     return tacoUtility.PromisesUtils.chain(Object.keys(components), (componentName: string) => {
                         // No override on the case of CLI project update - Cordova CLI gets its pinned version
                         var componentOverride: string = editParams.isKitProject ? componentName + "@" + components[componentName] : componentName;
+                        // call cordova helper to read config.xml
```

Comment:
Is this comment correct? It looks like you're just adding the plugins, not reading config.xml?